### PR TITLE
Conversion:Survey related fragments and activities

### PR DESCRIPTION
**Pull request by robustTechie**
Fixes #Issue_Number

Converted the following classes which were previously in Java, to Kotlin -
1. SurveyQuestionActivity
2. SurveyQuestionFragment
3. SurveySubmitFragment
4. SurveySubmitMvpView
5. SurveySubmitPresenter
6. SurveyListFragment
7. SurveyListMvpView
8. SurveyListPresenter

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Wed Aug 19 12:37:20 IST 2020
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```
Comment (Grandolf49) [inline mifosng-android/gradle/wrapper/gradle-wrapper.properties:6]: @robustTechie I think we'll need to remove these files from git under `mifosng-android/gradle*`

File: mifosng-android/local.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Wed Aug 19 12:37:09 IST 2020
+sdk.dir=/home/shashank/Android/Sdk
```
Comment (Grandolf49) [inline mifosng-android/local.properties:8]: This one as well

File: mifosng-android/gradlew [hunk 3]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (Grandolf49) [inline mifosng-android/gradlew:1]: This one as well

File: mifosng-android/gradlew.bat [hunk 4]
```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (Grandolf49) [inline mifosng-android/gradlew.bat:1]: This one too

File: mifosng-android/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (robustTechie) [reply mifosng-android/gradle/wrapper/gradle-wrapper.properties]: Done

File: mifosng-android/local.properties (hunk 2 above)
Comment (robustTechie) [reply mifosng-android/local.properties]: Done

File: mifosng-android/gradlew.bat (hunk 4 above)
Comment (robustTechie) [reply mifosng-android/gradlew.bat]: Done

File: mifosng-android/gradlew (hunk 3 above)
Comment (robustTechie) [reply mifosng-android/gradlew]: Done

**Review by Grandolf49 — CHANGES_REQUESTED** [review]
@robustTechie We'll need to remove the `mifosng-android/gradle/wrapper/gradle-wrapper.jar` as well!

**Comment by robustTechie** [comment]
> @robustTechie We'll need to remove the `mifosng-android/gradle/wrapper/gradle-wrapper.jar` as well!

Done

**Review by Grandolf49 — APPROVED** [review]
LGTM

**Pull request merged**